director/sort.go

```diff
@@ -144,21 +148,41 @@ func checkOverrides(addr net.IP) (coordinate *Coordinate) {
 	return nil
 }
 
-func getLatLong(addr netip.Addr) (lat float64, long float64, err error) {
+func getLatLong(ctx context.Context, addr netip.Addr) (lat float64, long float64, err error) {
 	ip := net.IP(addr.AsSlice())
 	override := checkOverrides(ip)
 	if override != nil {
 		log.Infof("Overriding Geolocation of detected IP (%s) to lat:long %f:%f based on configured overrides", ip.String(), (override.Lat), override.Long)
 		return override.Lat, override.Long, nil
 	}
 
+	network, ok := utils.ApplyIPMask(addr.String())
+	if !ok {
+		log.Warningf("Failed to apply IP mask to address %s", ip.String())
+	}
+
+	project, ok := ctx.Value(ProjectContextKey{}).(string)
+	if !ok {
+		log.Warningf("Failed to get project from context")
+		project = "unknown"
```

Comment:
The function `extractProjectFromUserAgent` returns an empty string if no project exists in the user agent. Does that get picked up here and assigned to "unknown", or does that need to happen explicitly? My hunch is it needs to be explicit: `if !ok || project == ""`